.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -1,12 +1,17 @@
 ---
 name: Proposal
-about: Termplate for proposed features to include in Interop 2022
+about: Termplate for proposed features to include in Interop 2023
```

Comment:
Do we want to move to [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)?